ci-operator/config/konflux-ci/e2e-tests/konflux-ci-e2e-tests-main.yaml

```diff
@@ -46,7 +46,7 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.14"
-  skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|^tests/release/pipelines/|^tests/load-tests/|^cmd/loadTests.go$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|PROJECT|LICENSE)$
```

Comment:
why was `release` removed from this pattern?

Reply:
because it was pointing to a subdirectory?

Reply:
hmm I assumed that we no longer need to skip triggering e2e jobs when `tests/release/pipelines/` test files are changed thanks to the rule engine Danny has implemented.

@jinqi7 can you confirm, or is it not ready yet and I should revert that change?

Reply:
If the test files in `tests/release/pipelines/` directory  are changed, we need to trigger e2e-tests jobs.

Reply:
thanks, so I'll keep it

Reply:
could you please add `/lgtm` if the change looks ok?